core/governance/engine.go

```diff
@@ -332,16 +334,16 @@ func (e *Engine) removeActiveProposalByID(ctx context.Context, id string) {
 	}
 }
 
-// OnTick triggers time bound state changes.
 func (e *Engine) OnTick(ctx context.Context, t time.Time) ([]*ToEnact, []*VoteClosed) {
+	now := t.Unix()
+
+	voteClosed, addToActiveProposals := e.evaluateBatchProposals(ctx, now)
```

Comment:
Given that they have just passed, I thought it might be useless to add them as they should not really need to be evaluated by all the code below (they were just closed). So it's ok to evaluate them in the next cycle - next ontick.